### ISSUE TITLE
docs: autodoc failure at rtfd.org

### ISSUE THREAD

**Issue by tiborsimko** [inveniosoftware/claimstore#36]
The doc building at rtfd.org does not generate auto-documentation properly:

- http://claimstore.readthedocs.org/en/latest/api.html#api

Locally, everything is fine. 

We should either (1) investigate and fix this failure at rtfd.org or (2) move documentation to pythonhosted.org.

**Comment by tiborsimko** [comment]
@lnielsen I think you saw the same troubles with Flask-AppFactory?

- http://flask-appfactory.readthedocs.org/en/latest/api.html

**Comment by tiborsimko** [comment]
OK, problem understood:

- need to build in their own virtual environments with CPython 3.x; however this lead to an error message `flask` not being installable...
- need to amend pre-requisite uppercase/lowercase (e.g. `Flask` instead of `flask`) so that `python setup.py install` would be working

**Comment by tiborsimko** [comment]
The setup step is now OK:

```
Finished processing dependencies for claimstore==0.1.0.dev20150821
```

but the autodoc step still fails:

```
Running Sphinx v1.3.1
making output directory...
loading translations [en]... done
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [readthedocs]: targets for 6 source files that are out of date
updating environment: 0 added, 1 changed, 0 removed
reading sources... [100%] api

/home/docs/checkouts/readthedocs.org/user_builds/claimstore/checkouts/latest/docs/api.rst:15: WARNING: autodoc: failed to import module 'claimstore.app'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/claimstore/envs/latest/local/lib/python3.4/site-packages/sphinx/ext/autodoc.py", line 385, in import_object
    __import__(self.modname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/claimstore/envs/latest/local/lib/python3.4/site-packages/claimstore-0.1.0.dev20150821-py3.4.egg/claimstore/app.py", line 26, in <module>
    from claimstore.core.exception import RestApiException
ImportError: No module named 'claimstore.core'
[...]
```
```

**Comment by tiborsimko** [comment]
Solved by a6d351973df8c607835b0b8b63c3d4aa8b961dd2:

```
diff --git a/docs/conf.py b/docs/conf.py
index a3900d3..9c05370 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -18,7 +18,7 @@ import os
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('..'))
```

(CC @jirikuncar @lnielsen @jbenito3 for information)
